### PR TITLE
Review EC book 6 NPCs (part 1 of 6)

### PR DESCRIPTION
**Pull request by roxim5**

### CODE REVIEW

File: packs/data/extinction-curse-bestiary.db/convergent-giant-eagle.json [hunk 1]
```diff
@@ -285,7 +289,8 @@
                         "key": "RollOption",
                         "label": "An ally with Convergent Tactics is within reach of the target",
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/convergent-giant-eagle.json:290]: Should be @Localized

Comment (roxim5) [reply packs/data/extinction-curse-bestiary.db/convergent-giant-eagle.json]: Localized

File: packs/data/extinction-curse-bestiary.db/saurian-warmonger.json [hunk 2]
```diff
@@ -1128,7 +1119,7 @@
                     "value": 2
                 },
                 "description": {
-                    "value": "<p>The saurian warmonger bellows, evoking the power of the primordial world.</p>\n<p>Non-dinosaur creatures within 30 feet take [[/r 10d6[sonic]]] damage (@Check[type:fortitude|dc:34|basic:true] save). On a failed save, a creature is also @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 2} for 1 round (@UUID[Compendium.pf2e.conditionitems.Stunned]{Stunned} [[/r 1d4 #stunned]]{1d4} on a critical failure).</p>\n<p>The saurian can't use Boneshaking Roar again for [[/br 1d4 #rounds]]{1d4 rounds}.</p>"
+                    "value": "<p>The saurian warmonger bellows, evoking the power of the primordial world.</p>\n<p>Non-dinosaur creatures within @Template[type:emanation|distance:30]{30 feet} take [[/r 10d6[sonic]]] damage (@Check[type:fortitude|dc:34|basic:true] save). On a failed save, a creature is also @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 2} for 1 round (@UUID[Compendium.pf2e.conditionitems.Stunned]{Stunned} [[/r 1d4 #stunned]]{1d4} on a critical failure).</p>\n<p>The saurian can't use Boneshaking Roar again for [[/br 1d4 #rounds]]{1d4 rounds}.</p>"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/saurian-warmonger.json:1122]:
```suggestion
                    "value": "<p>The saurian warmonger bellows, evoking the power of the primordial world.</p>\n<p>Non-dinosaur creatures within @Template[type:emanation|distance:30]{30 feet} take [[/r 10d6[sonic]]] damage (@Check[type:fortitude|dc:34|basic:true] save). On a failed save, a creature is also @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 2} for 1 round (@UUID[Compendium.pf2e.conditionitems.Stunned]{Stunned} [[/r 1d4 #Stunned Value]]{1d4} on a critical failure).</p>\n<p>The saurian can't use Boneshaking Roar again for [[/br 1d4 #Boneshaking Roar Recharge]]{1d4 rounds}.</p>"
```

File: packs/data/extinction-curse-bestiary.db/saurian-warmonger.json [hunk 3]
```diff
@@ -545,22 +530,27 @@
...
-            "_id": "Bi3LqfePnFUwEfYT",
-            "img": "systems/pf2e/icons/equipment/adventuring-gear/net.webp",
+            "_id": "hVW4wdrXb5dxMb9c",
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.equipment-srd.grmaV4GdoGD7sKbn"
+                }
+            },
+            "img": "systems/pf2e/icons/equipment/weapons/filchers-fork.webp",
             "name": "+1 Striking Mancatcher",
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/saurian-warmonger.json:540]: Why did you use a Scimitar as the base item for this?

File: packs/data/extinction-curse-bestiary.db/saurian-warmonger.json [hunk 4]
```diff
@@ -637,21 +629,21 @@
                     "custom": "",
                     "rarity": "common",
                     "value": [
-                        "evocation",
-                        "magical"
+                        "forceful",
+                        "sweep"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/saurian-warmonger.json:633]: Where do these come from? The traits aren't listed on the attack.

File: packs/data/extinction-curse-bestiary.db/saurian-warmonger.json [hunk 5]
```diff
@@ -545,22 +530,27 @@
...
         },
         {
-            "_id": "Bi3LqfePnFUwEfYT",
-            "img": "systems/pf2e/icons/equipment/adventuring-gear/net.webp",
+            "_id": "hVW4wdrXb5dxMb9c",
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.equipment-srd.grmaV4GdoGD7sKbn"
+                }
+            },
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/saurian-warmonger.json:538]: Should be removed as a Mancatcher isn't a Scimitar

File: packs/data/extinction-curse-bestiary.db/saurian-worldwatcher.json [hunk 6]
```diff
@@ -1514,7 +1499,7 @@
...
                 },
                 "description": {
-                    "value": "<p>The saurian worldwatcher bellows, evoking the power of the primordial world.</p>\n<p>Non-dinosaur creatures within 30 feet take [[/r 10d6[sonic]]] damage (@Check[type:fortitude|dc:38|basic:true] save). On a failed save, a creature is also @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 2} for 1 round (@UUID[Compendium.pf2e.conditionitems.Stunned]{Stunned} [[/r 1d4 #stunned]]{1d4} on a critical failure).</p>\n<p>The saurian can't use Boneshaking Roar again for [[/br 1d4 #rounds]]{1d4 rounds}.</p>"
+                    "value": "<p>The saurian warmonger bellows, evoking the power of the primordial world.</p>\n<p>Non-dinosaur creatures within @Template[type:emanation|distance:30]{30 feet} take [[/r 10d6[sonic]]] damage (@Check[type:fortitude|dc:38|basic:true] save). On a failed save, a creature is also @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 2} for 1 round (@UUID[Compendium.pf2e.conditionitems.Stunned]{Stunned} [[/r 1d4 #stunned]]{1d4} on a critical failure).</p>\n<p>The saurian can't use Boneshaking Roar again for [[/br 1d4 #rounds]]{1d4 rounds}.</p>"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/saurian-worldwatcher.json:1502]:
```suggestion
                    "value": "<p>The saurian worldwatcher bellows, evoking the power of the primordial world.</p>\n<p>Non-dinosaur creatures within @Template[type:emanation|distance:30]{30 feet} take [[/r 10d6[sonic]]] damage (@Check[type:fortitude|dc:38|basic:true] save). On a failed save, a creature is also @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 2} for 1 round (@UUID[Compendium.pf2e.conditionitems.Stunned]{Stunned} [[/r 1d4 #Stunned Value]]{1d4} on a critical failure).</p>\n<p>The saurian can't use Boneshaking Roar again for [[/br 1d4 #Boneshaking Roar Cooldown]]{1d4 rounds}.</p>"
```

**Review by SpartanCPA — CHANGES_REQUESTED** [review]

**Comment by SpartanCPA** [comment]
MIght as well localize the convergent tactics for Kendley Nathrael as well in this PR. Will require a rebase to do so.

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**